final/server/dist/src/types.js

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
Otherwise when compiled into JS in `dist`, it captures ALL files.. even ones I don't want it o